geometry_msgs/Point cartesian_position
float64 gripper_pitch
float64 gripper_roll
string frame_id
